src/app/component/quiz/quiz.component.scss

```diff
@@ -1,59 +1,105 @@
+.quiz_card {
+  height: auto;
+}
 
-.card{
-    margin: 0 auto; 
-    margin-bottom: 50px;
-    border-radius: 15px;
-    height: auto;
-    
-}
-.card-body{
+.header_txt {
+  position: relative;
+  z-index: 2;
+  height: 70px;
+  padding: 0 15px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  .header_title {
+    font-size: 25px;
+    font-weight: 600;
+    user-select: none;
+  }
+}
 
-    padding: 20px 60px 60px;
-    margin: 8px;
-    display: inline-block;
+.timer {
+  color: #004085;
+  background: #cce5ff;
+  border: 1px solid #b8daff;
+  height: 45px;
+  width: 145px;
+  padding: 0px 8px;
+  border-radius: 5px;
+  display: flex;
```

Comment:
same please manage this with boostrap class

Reply:
yes, i managed it with boostrap class.